docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+
+### Networking
+The following components help harden the Base against networking attacks:
+
+* **No web interface**: Modern web applications, built conveniently with frameworks like Node.js or Flask, have hundreds of dependencies that are outside our control and could contain security vulnerabilities or even malicious code. This is why we decided not to provide a web interface directly on the Base and expose only a single API by default.
+
+* **NGINX**: although the Middleware exposes only a single API, additional components like electrs can optionally provide their own public API, e.g. for direct usage with the Electrum wallet. To have only a single entry-point from the public network into the Base, all communication is routed through [NGINX](https://www.nginx.com), which is a enterprise-grade, reliable and secure reverse proxy server.
+
+* **Firewall**: to mitigate against network snooping and the exploitation of potentially open ports on operating system level, very restrictive packet filtering rules are set with [iptables](https://netfilter.org/projects/iptables/index.html) to refuse any connections that are not explicitely allowed.
+
+* **Brute-force protection**: by default, SSH-login is disabled. Once enabled, only login with SSH keys should be used. Nonetheless, we use [fail2ban](https://www.fail2ban.org) to log any login attempts and block ip addresses for a certain time after too many unsuccessful tries.
+
+* **SSH Keys**: for secure usage of SSH, the password login is disabled by default, so that a terminal login is only possible using SSH keys. The public key can be integrated in the Base image on build time, or later with the App.
+
+* **Disabled root user**: SSH login for the root user is disabled by default, even using SSH keys. This way, even when gaining access to the device using a stolen key, the attacker still needs to enter the root password when using `sudo`.
+
+* **Service users**: every service runs as a dedicated service user, with as little privileges as possible.
+
+### Local security
+A network appliance like the BitBox Base, built on readily available components, can never be assumed to be safe against an attacker that has physical access to the device. Nonetheless, device should be safe against casual tampering.
+
+* **Tamper-evident opening**: by providing support for tamper-evident seals, the device can be secured in a way that makes it evident if it has been opened.
+
+* **Disabled users**: both `root` and `base` users are locked by default, and can only be unlocked using the BitBox App over the secure API after setting a new password.
+
+* **Disabled USB**: to prevent tampering with a USB keyboard, the USB ports are configured to ignore USB-HID by default.
+
+This list of security measures is non-exaustive and meant to grow over time. Please let us know by opening a [GitHub issue](https://github.com/digitalbitbox/bitbox-base/issues) if you think we are missing a critical security feature!
```

Comment:
Should we really be asking reviewers who find that critical security features are missing to open public issues? Would it make sense to point them to the responsible disclosure process described at https://shiftcrypto.ch/bug-bounty-program/?

Also, should we consider adding a `SECURITY.md` to this repo, which recently got quite good Github UI features to support it?

* https://github.com/digitalbitbox/bitbox-base/security/policy
* https://snyk.io/blog/ten-git-hub-security-best-practices/

Reply:
I rephrased that ideas for additional security controls (features) should be documented as public issues, but security issues should use our bounty program.

Reply:
Re: SECURITY.md: I suggested that for the BitBox02 last week and I think we can wait for it to be implemented there and then adapt it. For the BitBox Base it's not really necessary at this early stage.

Reply:
> I rephrased that ideas for additional security controls (features) should be documented as public issues, but security issues should use our bounty program.

Where are we saying that security issues should use the bounty program? As of 7d614a343bdb92a76a73c9244e94b23e536227aa, I'm still only seeing the text below:

> This list of security controls is non-exhaustive and meant to grow over time. Please let us know by opening a [GitHub issue](https://github.com/digitalbitbox/bitbox-base/issues) if you think we are missing a critical security feature!

Reply:
Okay, I'm seeing the changes now. Closing the thread.